api/models.py

```diff
@@ -161,16 +161,15 @@ class ActivityType(Base):
     __tablename__ = 'activitiyTypes'
```

Comment:
I am wondering is it possible to correct this spelling error on the table name once and for all so it doesn't give us a problem in the future?

Reply:
I'm on it

Reply:
LGTM. After you've made final changes let me know so I can merge.